backend/src/apiserver/ilab_pipeline/pipeline.yaml

```diff
@@ -0,0 +1,2326 @@
+# PIPELINE DEFINITION
```

Comment:
lets put this file in `backend/src/apiserver/sample_pipelines`, or something similar, so we can expand on this later.

Also, we should name this file something a little more clear than `pipeline.yaml`.  Perhaps `instructlab.yaml`?